openapi/data_repository_service.swagger.yaml

```diff
@@ -194,57 +194,71 @@ definitions:
           sha512
   Bundle:
     type: object
-    required: ['id', 'object_ids', 'created', 'updated', 'version', 'checksums']
     properties:
       id:
         type: string
-        description: |-
+        description: >-
           An identifier, unique to this Data Bundle
       object_ids:
         type: array
         items:
           type: string
-        description: |-
+        description: >-
           The list of Data Objects that this Data Bundle contains.
       created:
         type: string
         format: date-time
-        description: |-
+        description: >-
           Timestamp of object creation in RFC3339.
       updated:
         type: string
         format: date-time
-        description: |-
-          Timestamp of update in RFC3339, identical to create timestamp in systems
-          that do not support updates.
+        description: >-
+          Timestamp of update in RFC3339, identical to create timestamp in
+          systems that do not support updates.
       version:
         type: string
-        description: |-
-          A string representing a version, some systems may use checksum, a RFC3339
-          timestamp, or incrementing version number. For systems that do not support
-          versioning please use your update timestamp as your version.
+        description: >-
+          A string representing a version, some systems may use checksum, a
+          RFC3339 timestamp, or incrementing version number. For systems that do not
+          support versioning please use your update timestamp as your version.
+      mime-type:
+        type: string
+        description: >-
+          A string providing the mime-type of the Data Bundle.
+          For example, "application/json".
       checksums:
         type: array
-        items:
-          $ref: '#/definitions/Checksum'
-        description: |-
+        description: >-
           At least one checksum must be provided.
           The Data Bundle checksum is computed over all the checksums of the
           Data Objects that bundle contains.
```

Comment:
Can we be more explicit about how to compute the checksum?
* Is the idea simple checksum string concatenation? E.g. if Bundle B1 contains Objects O1 and O2, with checksums "cs111" and "cs222", then do I compute the checksum of bundle B1 by running my hash function on "cs111cs222"?
* what order should I use in computing my checksum? E.g. if Bundle B2 contains the same two objects, but happens to return them in a different order in the `contents` array, does it have a different checksum? 
* does the checksum include the `name` field from `BundleObjects`? If not, are we okay with two bundles that return differently named objects having the same checksum, as long as the objects have the same contents?

Reply:
Description updated. The algorithm is based on a data bundle checksum is based on a concatenated alphabetically (hex-code) sorted checksum of data objects